pkg/deployments/README.md

```diff
@@ -66,7 +66,7 @@ Returns an object with all contracts from a deployment and their addresses.
 ## Active Deployments
 
 | Description                                            | Task ID                                                                                              |
-| ------------------------------------------------------ | ---------------------------------------------------------------------------------------------------- |
```

Comment:
Make sure this part doesn't change, because it can conflict with other deployments PRs